### DISCUSSION TITLE
OIDCXForwardedHeaders with Forwarded header

### DISCUSSION THREAD

**Discussion by studersi** [OpenIDC/mod_auth_openidc#853] [Ideas]
The new [`OIDCXForwardedHeaders`](https://github.com/zmartzone/mod_auth_openidc/blob/master/auth_openidc.conf#L969) directive works as expected with the relevant `X-Forwarded-*` headers. These headers are generally available and the de facto standard.

However, there is also the standardized [`Forwarded`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded) header, which is not supported by `OIDCXForwardedHeaders`:
```
Invalid value for directive 'OIDCXForwardedHeaders': invalid value 'Forwarded', must be one of ['X-Forwarded-Host'|'X-Forwarded-Port'|'X-Forwarded-Proto']
```

I do not see any particular problems with this but thought I might mention it anyway. Maybe newer technologies might prefer to set the standardized header instead of the `X-Forwarded-*` headers.

**Comment by zandbelt** [comment]
yes, that's a good point and actually one of the reasons for making it a configurable primitive; I'll take a look "soonish" but a PR is also welcome

**Reply by zandbelt** [reply]
see https://github.com/zmartzone/mod_auth_openidc/commit/1a5e2a908b6913adf6694226b76398642d6da2ab

**Reply by studersi** [reply]
Wow that was super quick, thank you!
